gapipy/resources/dossier/activity_dossier.py

```diff
@@ -23,3 +24,42 @@ class ActivityDossier(Resource):
         ('start_location', 'Place'),
         ('end_location', 'Place'),
     ]
+
+    _model_collection_fields = [
+        ('details', DossierDetail),
+    ]
+
+    def __repr__(self):
+        return '<{} {}>'.format(self.__class__.__name__, self.name)
+
+    @property
+    def duration(self):
+        """
+        Simulate the duration dict in `ItineraryComponent` objects,
```

Comment:
I see the benefit here, but do we want to simulate this to be "dot-accessible" like most objects in gapipy?